src/Symfony/Component/HttpKernel/Tests/DataCollector/DataCollectorTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Tests\Fixtures\DataCollector\CloneVarDataCollector;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Test\VarDumperTestTrait;
+
+class DataCollectorTest extends \PHPUnit_Framework_TestCase
+{
+    use VarDumperTestTrait;
```

Comment:
not sure I'd use this trait
instead, I propose to turn the Data object into a string using:
```php
        $dumper = new CliDumper();
        $dumper->setColors(false);
        $this->assertDumpMatchesFormat($expected, $dumper->dump($data, true));
```